### PR TITLE
[IoT] Enabled minimum TLS version on IoT Hub Create

### PR DESCRIPTION
**Pull request by c-ryan-k**
**Description**  

Enables users to create IoT Hubs through the CLI with a minTLS version (currently only "1.2") set.
**Testing Guide**  

To create a hub that enforces TLS version 1.2:
`az iot hub create -n {hub_name} -g {resource_group} -l {location} --sku {sku} --min-tls-version 1.2`
**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by c-ryan-k** [comment]
@yonzhan / @zhoxing-ms - anything holding this back from being reviewed / merged?

**Comment by yonzhan** [comment]
@zhoxing-ms please help with this PR.

File: src/azure-cli/azure/cli/command_modules/iot/_params.py [hunk 1]
```diff
@@ -175,6 +175,9 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
                    type=int, validator=validate_fileupload_sas_ttl,
                    help='The amount of time a SAS URI generated by IoT Hub is valid before it expires,'
                         ' between 1 and 24 hours.')
+        c.argument('min_tls_version', options_list=['--min-tls-version', '--mintls'],
+                   type=str, help='Specifies the minimum TLS version to support for this hub. Can be set to'
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/iot/_params.py:179]: Please write help in the first person voice
```suggestion
                   type=str, help='Specify the minimum TLS version to support for this hub. Can be set to'
```

**Comment by c-ryan-k** [comment]
@zhoxing-ms - after that string change, looks like CI failed due to timeout loading webapp module - can we re-run the CI please?

**Review by digimaun — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Comment by c-ryan-k** [comment]
@zhoxing-ms - anything specific holding this back from being merged?

**Pull request merged**